packages/cli/src/commands/create/index.ts

```diff
@@ -47,13 +51,21 @@ export const createCommand = program
   .option('-b, --blueprint <blueprint>', '[DEPRECATED] use --template instead')
   .option('--skip-space', 'skip space creation')
   .option('--token <token>', 'Storyblok access token (skip space creation and use this token)')
+  .option(
+    '-r, --region <region>',
+    `The region to configure for your SDK. This flag is used for space creation when applicable and is also applied to the generated project template.`,
+  )
   .action(async (projectPath: string, options: CreateOptions) => {
     konsola.title(`${commands.CREATE}`, colorPalette.CREATE);
     // Global options
     const verbose = program.opts().verbose;
     // Command options - handle backward compatibility
     const { template, blueprint, token } = options;
 
+    if (options.region && !isRegion(options.region)) {
+      handleError(new CommandError(`The provided region: ${options.region} is not valid. Please use one of the following values: ${Object.values(regions).join(' | ')}`));
+      return;
+    }
```

Comment:
I think this error message is super helpful in case people enter an invalid region!